skills/snip-36/SKILL.md

```
@@ -0,0 +1,488 @@
+allowed-tools:
+  - Bash
+  - Read
+  - Write
+  - Glob
+  - Grep
+  - Task
+user-invocable: true
+---
+
+
+## Overview
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)